.github/workflows/build_dependabot_bundler_pr.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow is based on https://github.com/dependabot/dependabot-actions-workflow
+
+name: Build Dependabot Bundler PR
+on:
+  push:
+    branches:
+      - "dependabot/bundler**"
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Golang
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.19
+
+      - name: run mod check
+        run: GOPATH=$(go env GOPATH) make -j $(nproc) mod.check
+
+      - run: mkdir ./dependabot-pr
+
+      - name: Save branch ref
+        run: echo ${{ github.ref }} | sed 's/refs\/heads\///' > ./dependabot-pr/BRANCH_REF
+
+      - name: Configure git
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+
+      # NOTE: Prefixing/appending commit messages with `[dependabot skip]` allows
+      # dependabot to rebase/update the pull request, force-pushing over any changes
+
+      - name: Commit and copy license changes
```

Comment:
Rook doesn't have `.licenses` so maybe we remove this? Or I guess it's harmless.